dai_cadcad/flip.py

```diff
@@ -0,0 +1,108 @@
+""" Flip Module
+    Class-based representation of the Flip smart contract
+    (contains only what is necessary for the simulation)
+"""
+
+
+class Flip:
+    ADDRESS = "flip"
```

Comment:
Let's instantiate `ADDRESS` in the `__init__` function - since we'll have one `Flipper` per Ilk type, we should probably make the address something like `f"flipper_{ilk}"`